content/docs/ethernet-v0.5/overview.md

```diff
@@ -0,0 +1,41 @@
++++
+Description = ""
+date = "2017-04-14T13:00:00+00:00"
+title = "Overview of the ethernet infrastructure"
+parent = "/docs/ethernet-v0.5/"
+next = "/docs/ethernet-v0.5/environment/"
+showdisqus = true
+
++++
+
+<a name="figure-overview"></a>
+<img src="../figures/lowRISC_tag.png" alt="Drawing" style="width: 600px; padding: 20px 0px;"/>
+
+## Pre-defined Design constraints
+
+Our goal of supporting the same FPGA board as the previous release
+may only be realised with a single 100MHz Ethernet port, due to PCB design constraints.
+Nevertheless it is conceived that 1GHz Ethernet could be made use of by a different board.
```

Comment:
Could you perhaps be a little clearer with this paragraph, given it is part of the high-level "overview". For the average reader, what does a "100MHz Ethernet port" mean?